raptorWeb/templates/raptormc/base.html

```diff
@@ -24,220 +23,163 @@
+                  <ul class="dropdown-menu bg-dark text-center">
+                    <li><a class="dropdown-item text-white" href="https://donate.shadowraptor.net/" target="_blank">Packages <img class="new_tab_icon" src="{% static 'image/new_tab_white.svg' %}" alt="Opens in new tab"></a></li>
+                    <li><a class="dropdown-item text-white" href="https://www.patreon.com/shadowraptornetwork" target="_blank">Patreon <img class="new_tab_icon" src="{% static 'image/new_tab_white.svg' %}" alt="Opens in new tab"></a></li>
+                  </ul>
+                </li>
+    
+                <li class="btn btn-secondary m-2 p-2 dropdown">
+                  <a class="nav-link dropdown-toggle text-white fs-5" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">More</a>
+                  <ul class="dropdown-menu bg-dark text-center">
+                    <li><a class="dropdown-item text-white" href={% url 'raptormc:staff_apps' %}>Staff Applications</a></li>
+                    <li><a class="dropdown-item text-white" href={% url 'raptormc:site_members' %}>Site Members</a></li>
+                    <li><hr class="dropdown-divider"></li>
+                    <li><a class="dropdown-item text-white" href="https://bans.shadowraptor.net/" target="_blank">Punishments <img class="new_tab_icon" src="{% static 'image/new_tab_white.svg' %}" alt="Opens in new tab"></a></li>
+                  </ul>
+                </li>
+    
+              </ul>
             </div>
           </div>
         </div>
-      </nav>
       
-      <nav class="navbar navbar-dark bg-dark">
-        <div class="container">
-          <div class="d-flex w-100">
+        <nav class="navbar navbar-dark bg-dark">
+          <div class="container">
             <div class="navbar-header d-flex">
-              <button id="navToggle" class="navbar-toggler m-3 mt-0 mb-0 border-2" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent">
-                <span class="navbar-toggler-icon" ></span>
-              </button>
-              <a class="navbar-brand m-3 mt-0 mb-0" href="{% url 'raptormc:home' %}">
-                <img id="navImage" class="w-75" src="{% static 'image/letters.webp' %}" alt="ShadowRaptor">
-              </a>
-            </div>
-            <div class="navbar-header d-flex w-25">
-              <div id="spacer" class="col-lg-2 col-0">
-                <span></span>
+              <div class="row p-2 pt-1 pb-1">
+                <a class="col-3 col-sm-2 col-lg-1 btn btn-outline-secondary" role="button" data-bs-toggle="tooltip" 
+                {% if discord_guild %}
+                data-bs-title="{{ discord_guild.online_members }}/{{ discord_guild.total_members }} Members Online" href="https://discord.gg/GEhyhku" target="_blank">
+                {% else %}
+                data-bs-title="No Guld" href="https://discord.com" target="_blank">
```

Comment:
Typo here, but we should just not show tooltip if there is no guild

Another note, the invite link here can be automatically created by the bot for the DiscordGuild model